specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json

```diff
@@ -538,7 +545,7 @@
           "200": {
             "description": "",
             "schema": {
-              "$ref": "#/definitions/ResourceListResult"
+              "$ref": "#/definitions/ResourceExpandedListResult"
```

Comment:
Changing of the type names in swagger will introduce unnecessary breaking changes in the SDKs of every language. If this is not absolutely required, I am suggesting to change this back, which will avoid breaking changes in most of the SDKs.

Reply:
Good point. Will do